packages/repository-json-schema/src/build-schema.ts

```diff
@@ -125,6 +126,10 @@ export function metaToJsonProperty(meta: PropertyDefinition): JSONSchema {
 
   if (isComplexType(propertyType)) {
     Object.assign(propDef, {$ref: `#/definitions/${propertyType.name}`});
+  } else if (propertyType === Date) {
+    Object.assign(propDef, {
+      type: 'date-time',
```

Comment:
```js
type: 'string', 
format: 'date-time',
```

Reply:
@hacksparrow before you fix this: please look around the tests and try to find a test that runs JSON Schema validation against the JSON/OpenAPI schema we are producing.

Ideally, if our schema builder emits an invalid schema (as you did here), then there should be a test that fails and reports the problem.

Reply:
Maybe we have an acceptance test that needs to be expanded with the newly supported types?

Reply:
Noted.